definitions/prometheus_exporters/prometheus_githubactions.yml

```diff
@@ -0,0 +1,29 @@
+provider: prometheus
+service: github
+display_name: Github
+entities:
+- name: job
```

Comment:
is this a unique entity? looks like we have to add some labels to form different entities from the labels